src/mono/wasm/debugger/DebuggerTestSuite/EvaluateOnCallFrameTests.cs

```diff
@@ -1016,6 +1016,28 @@ public async Task EvaluateLocalObjectFromAssemblyNotRelatedButLoaded()
                Assert.Equal(internalAndProtected[1]["value"]["value"], "protected");
                Assert.Equal(priv[0]["value"]["value"], "private");
            });
+        
+        [Fact]
+        public async Task StructureGetters() =>  await CheckInspectLocalsAtBreakpointSite(
```

Comment:
Look at the other tests that use getters (grep for `TGetter`), and add values to those. The second parameter for that can be the value. For example: `TGetter("foo", TString("value"))`

Reply:
.. those should be working.

Reply:
I created a new refactoring [issue](https://github.com/dotnet/runtime/issues/63667) about this case and a similar one we discussed on structure properties separation PR, to remove some dedicated test classes.

Reply:
About just putting a value there. I'm pretty sure it works only for simple getters (autoexpandable), without a custom return. In case I'm wrong then I will exchange the mechanism with your suggestion while refactoring.

Reply:
Having the value there also will check the value returned by the getter, which can be affected by fields/properties getting overriden, or shadowed.